src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ContextQuery/CSharpSyntaxContext.cs

```diff
@@ -76,11 +76,12 @@ internal sealed class CSharpSyntaxContext : SyntaxContext
             bool isGenericTypeArgumentContext,
             bool isGlobalStatementContext,
             bool isImplicitOrExplicitOperatorTypeContext,
+            bool isInheritanceRequiringClassContext,
+            bool isInheritanceRequiringInterfaceContext,
```

Comment:
IsBaseClassContext (and the same for interface)

Reply:
IsBaseInterfaceContext or what?

Reply:
That sounds fine to me.